packages/application-to-register-webapp/client/js/pages/land/add-legal-agreement-parties.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-env jquery */
+$(document).ready(function () {
+  if (window.location.pathname === '/land/add-legal-agreement-parties') {
+    $('.govuk-body').bind('DOMSubtreeModified', function () {
+      $('.govuk-radios__item').click((event) => {
+        if ($(event.target.parentNode).next('div').hasClass('govuk-radios__conditional--hidden')) {
+          $(event.target.parentNode).next('div').removeClass('govuk-radios__conditional--hidden')
+        } else {
+          $(event.target.parentNode).parent().show().find('.govuk-radios__conditional').addClass('govuk-radios__conditional--hidden')
+        }
+      })
+    })
+  }
+})
```

Comment:
Is this file supposed to be referenced from [packages/application-to-register-webapp/src/views/land/add-legal-agreement-parties.html](https://github.com/DEFRA/biodiversity-net-gain-service/blob/a471b0c0806c3658c3e62e598636980effbf6439/packages/application-to-register-webapp/src/views/land/add-legal-agreement-parties.html)?.

If so please make corresponding changes to the HTML file and [packages/application-to-register-webapp/webpack.config.js](https://github.com/DEFRA/biodiversity-net-gain-service/blob/master/packages/application-to-register-webapp/webpack.config.js)